interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md

```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
 
 ![WinFormsPropertyGrid-ResetValue](../../../../../images/eurd-win-FormsPropertyGrid-ResetValue.png)
 
+## Set Color Properties
+
+You can use Magnifier to set color properties.
+
+![](../../../../../images/eurd-win-magnifier-button.png)
+
+Do the following to use Magnifier:
+
+- Click the Magnifier button.
+- Move the invoked Magnifier on the screen to find the color you want to set.
+
+	![](../../../../../images/eurd-win-magnifier-get-color.png)
+
+	You can zoom out the magnified zone. To do this use one of the following options:
+	
+	- Hold Ctrl and press + / -
+	- Scroll the mouse wheel
+
+- Click to set the color property to the found color.
```

Comment:
What do you mean by "found color"? (This does not sound correct.)

Reply:
Changed to "Click to set the color property to the selected color."

Reply:
Thank you